<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма веса металла из системы:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_weightSys">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>371</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма веса металла после отправки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_weight">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_weightDiff">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>270</y>
     <width>621</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_goBack">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>640</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 153, 153);
color: rgb(254, 251, 251);</string>
   </property>
   <property name="text">
    <string>На главную</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_showCheque">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Показать чек</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
